pkg/status/status.go

```diff
@@ -124,7 +124,7 @@ func (s *Service) LocalSnapshot() (*Snapshot, error) {
 		PullsyncRate:            syncRate,
 		StorageRadius:           uint32(storageRadius),
 		ConnectedPeers:          connectedPeers,
-		NeighborhoodSize:        neighborhoodSize,
+		NeighborhoodSize:        neighborhoodSize + 1, // include self
```

Comment:
This is for point 6 from https://github.com/ethersphere/bee/issues/461?

Reply:
yes